pkg/syncer/metadatasyncer.go

```diff
@@ -168,6 +170,22 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 		metadataSyncer.volumeManager = volumes.GetManager(ctx, vCenter)
 	}
 
+	if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
+		if metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.CSISVFeatureStateReplication) {
+			log.Info("Starting SvFSSReplicationService")
```

Comment:
You can move this to Line: 181

Reply:
actually moved it inside the function StartSvFSSReplicationService

Reply:
moved it inside - StartSvFSSReplicationService func